x-pack/filebeat/cmd/root.go

```diff
@@ -50,8 +50,8 @@ func defaultProcessors() []mapstr.M {
 				"when.not.contains.tags": "forwarded",
 			},
 		},
-		{"add_cloud_metadata": nil},
-		{"add_docker_metadata": nil},
-		{"add_kubernetes_metadata": nil},
+		// {"add_cloud_metadata": nil},
```

Comment:
Yes it definitely would be. We also don't want to duplicate them in the shipper Filebeat. The solution we should pursue for now should be to configure the shipper instance of Filebeat without any default processors, and leave them in the inputs. This will leave the processor configuration unchanged from what it is today.

In the near future we want all the default processors to migrate to the shipper (see https://github.com/elastic/elastic-agent-shipper/issues/292). There is some complexity to this however:
1. If someone disables the shipper the default processors need to go back in the inputs
2. The default processor configurations vary somewhat by Beat
3. The shipper's default processors are true global processors and will apply to events the did not before, like endpoint security.

Those are my three reasons for not wanting these enabled in the shipper Filebeat as part of this change.

Reply:
Ah, sorry, I did that while I was debugging to make the output a little less cluttered, forgot to remove it, hahaha

Reply:
I don't think it's enough to just put it back, I think you need to conditionally remove it when Filebeat is run as the shipper. Otherwise the default Filebeat processors will become the default output processors, which might have unexpected side effects. The default processors for each Beat are not all the same today.

Reply:
Also agreed; we should probably just have the shipper disable any global processors until we figure out how to move them into the shipper properly.